lib/features/digital_guide_view/presentation/widgets/expansion_tiles_content/surroundings_expansion_tile_content.dart

```diff
@@ -0,0 +1,49 @@
+import "package:flutter/material.dart";
+import "package:flutter_riverpod/flutter_riverpod.dart";
+
+import "../../../../../utils/context_extensions.dart";
+import "../../../data/models/digital_guide_response.dart";
+import "../../../data/models/surrounding_response.dart";
+import "../../../data/repository/surrounding_repository.dart";
+
+class SurroundingsExpansionTileContent extends ConsumerWidget {
+  const SurroundingsExpansionTileContent({
+    required this.digitalGuideResponse,
+  });
+
+  final DigitalGuideResponse digitalGuideResponse;
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final asyncSurroundingData = ref
+        .watch(getSurroundingDataProvider(digitalGuideResponse.surroundingId));
+
+    return asyncSurroundingData.when(
+      data: (surroundingData) => _SurroundingExpansionTileContent(
+          surroundingResponse: surroundingData),
+      // To-do nice error screen
```

Comment:
you have a lot of "todo" comments in you code. I would opt for using the "flutter style" (tbh is pretty universal style) for them: https://dart.dev/tools/linter-rules/flutter_style_todos

Reply:
Good advice